app.py

```diff
@@ -53,35 +53,10 @@
                 "Please see the following link http://nameofwebsite/help",
                 "Task failed successfully",
                 "Task successfully failed",
-                "big error, much problem",
-                """
-<html><body>
```

Comment:
I liked this error 😢

Reply:
Was confusing, especially if someone is new to cURL and might think that is a correct output